### DISCUSSION TITLE
Will Github tokens be passed to clients?

### DISCUSSION THREAD

**Discussion by barnettZQG** [dexidp/dex#2423] [Q&A] [answered]
I'm sorry and I have a simple question If we use the Github provider,  Can the client take the Token and communicate directly with Github OpenAPI?

**Comment by nabokihms** [comment, marked as answer]
Hello @barnettZQG. Fortunately, not. Users only receive Dex tokens that can be used to communicate with Dex only. 

Dex internally stores GitHub tokens for refreshing ID tokens, and users have no access to them.

**Reply by barnettZQG** [reply]
Thanks!
